utils/templates.py

```diff
@@ -261,3 +261,55 @@ def __init__(self, params=None, content=None):
 
         self.body = copy.deepcopy(self.params)
         self.body.update({'content': self.content})
+
+class ShortReviewReply(InvitationReply):
+    """
```

Comment:
Ok for now.  
On the other hand, if we want to make this accessible to others, we shouldn't have to rewrite invitations from scratch.  Maybe this submissions_done (or create_invitation) script belongs in admin/conference-template.